src/components/space/charts/ActionsByWeekdayChart.js

```diff
@@ -0,0 +1,63 @@
+import { Bar, BarChart, CartesianGrid, XAxis, YAxis } from 'recharts';
+
+import React, { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  findYAxisMax,
+  formatActionsByWeekday,
+  getActionsByWeekday,
+} from '../../../utils/api';
+import { filterActions } from '../../../utils/array';
+import ChartContainer from '../../common/ChartContainer';
+import ChartTitle from '../../common/ChartTitle';
+import { DataContext } from '../../context/DataProvider';
+import EmptyChart from './EmptyChart';
+
+const ActionsByWeekdayChart = () => {
+  const { t } = useTranslation();
+  const { actions, allMembers, selectedUsers, selectedActions } =
+    useContext(DataContext);
+
+  // ActionsByWeekday is the object passed, after formatting, to the BarChart component below
+
+  // if you remove all names in the react-select dropdown, selectedUsers becomes null
+  // if no users are selected (i.e. selectedUsers.size === 0), show all actions
+  // if all users are selected (i.e. selectedUsers.size === allMembers.size), also show all actions
+  // third condition above is necessary: some actions are made by users NOT in the users list (e.g. user account deleted)
+  // e.g. we retrieve 100 total actions and 10 users, but these 10 users have only made 90 actions
+  // therefore, to avoid confusion: when all users are selected, show all actions
+  let ActionsByWeekday = [];
```

Comment:
Variables don't have a capital as first letter (it might be false in the other part of the code you didn't write).